statsmodels/stats/libqsturng/qsturng_.py

```diff
@@ -474,13 +475,18 @@ def _func(a, p, r, v):
     if r == 3:
         f += -0.002 / (1. + 12. * _phi(p)**2)
 
+        if np.isinf(v):
```

Comment:
v=min(v,1e38)?

Reply:
Or maybe a ternary v = v is not isinf(v) else 1e38

Reply:
I tend to like the two-line version here because I can see in coverage reports which cases are hit.  Failing that I think the `v = min(v, 1e38)` would be my preference, which I think is what you used.